docs/css/style.css

```diff
@@ -95,7 +95,7 @@ nav.badges a + a {
 
 #_backers a img {
   width: 64px;
-  background: url(/images/backer-background.svg?inline) center center no-repeat;
+  background: url(/images/backer-background.svg) center center no-repeat;
```

Comment:
The inlining of this background gives a significantly improved user experience in the page rendering progress: http://webpagetest.org/video/compare.php?tests=190303_XT_8d430aac89f487867e1c34a2091db20c-r:1-c:0,190303_R9_221121187cf891367ca68538dc27e542-r:1-c:0